src/App.tsx

```diff
@@ -2,15 +2,29 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { useState } from 'react';
+
+type Movie = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+};
 
 export const App = () => {
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+  const goAddMovie = (movie: Movie) => {
+    setMovies(prevMovies => [...prevMovies, movie]);
+  };
```

Comment:
Handler names traditionally start on "on": 
```suggestion
  const onAddMovie = (movie: Movie) => {
    setMovies(prevMovies => [...prevMovies, movie]);
  };
```